app/src/main/java/com/infomaniak/drive/data/api/UploadTask.kt

```diff
@@ -186,6 +193,14 @@ class UploadTask(
         onFinish(uri)
     }
 
+    private suspend fun launchTaskEmptyFile() = withContext(Dispatchers.IO) {
+        callApi<ApiResponse<File>>(
+            uploadFile.uploadEmptyFileUrl(),
+            POST,
```

Comment:
Put back `ApiMethod.`.
We import everything except the last class name.

That's why I said to _"Import `ApiController.`"_.